### PR TITLE
GEN-1385 | Implement header for Widget Product Selector page

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- See title.

## Justify why they are needed

- I implemented it inline but it will likely end up as some sort of layout we can reuse on other pages in the flow.

- Exported grid layout styles so we can use them for more complex layouts

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3516** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3516" target="_blank">Graphite</a> 
    * **PR #3517** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3517" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3517?utm_source=stack-comment).

File: apps/store/src/pages/widget/[flow]/[shopSessionId]/select.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <div>
+      <Header as="header">
+        <LogoArea>
+          <HedvigLogo />
+        </LogoArea>
+
+        <ProgressArea>
+          <ProgressIndicator.Root>
+            <ProgressIndicator.Step active={true}>Your info</ProgressIndicator.Step>
```
Comment (alebedev) [inline apps/store/src/pages/widget/[flow]/[shopSessionId]/select.tsx:17]: Do we have some task for localizing widget UI later?

**Review by alebedev — APPROVED** [review]
Nice!

File: apps/store/src/pages/widget/[flow]/[shopSessionId]/select.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/pages/widget/[flow]/[shopSessionId]/select.tsx:17]: I can add one, yes :)

**Comment by robinandeer** [comment]
### Merge activity

* **Nov 6, 3:45 PM**: @robinandeer started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3517).

**Pull request merged**